.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --max-complexity 10 --max-line-length 127 --statistics
+        flake8 . --count --ignore=E203 --max-complexity 10 --max-line-length 127 --statistics
```

Comment:
flake8 and black checks disagree about spaces before collons. This disables the flake8 check and uses black style